### ISSUE TITLE
Plugin needs configuration bootstrap (for connecting to conveyor node)

### ISSUE THREAD

**Issue by milutz** [samtecspg/conveyor#142]
As of Conveyor 1.3 the only way to configure where the conveyor-node (API + node_red) is is by editing the kibana.yml.  The Conveyor plugin should allow the user to set the conveyor-node url in the UI any time the plugin can't talk to the API (or "any time" via a configuration UI).

This is highly beneficial because it would (particularly if the UI linked to conveyor-node install instructions) allow us to point people at the plugin binary alone, and it in turn would explain to the user how to get/install the conveyor-node.

This system should be version-aware (so dependent on issue #129) and request/require-you-change the URL if you try to connect to the incorrect version of the API.

**Comment by wrathagom** [comment]
so essentially we're going to move the conveyor node url and connection info into a conveyor configuration index.

On start we check for a value, if the value is null:
- we try the default dockerfile info
- we try the default localhost info

If either of those work we save the value into the configuration index. if neither work we have a screen that instructs the user on what to do.

If we find a value we use it. If it doesn't work we start over again.


-

**Comment by milutz** [comment]
@malave Does your solution for this implement the stuff that @wrathagom listed above? (first check docker env-vars, if that fails try localhost, if neither work pop up screen to install and configure)

If so, in what circumstance?

I just tried starting up, killing the conveyor-node and then clicking on conveyor in the kibana-UI.  I got the following:

![screenshot](https://user-images.githubusercontent.com/3475505/41252742-0f6f0278-6d8c-11e8-9fe3-0883042996a6.png)

This isn't a half-bad screen, but not what I expect if you implemented Calebs logic, and it makes me wonder if Caleb's logic is implemented at all

Assuming my tests are not missing functionality that you have implemented, do you want me to open a new issue for the functionality not implemented or do you want to reopen this one?
